arbnode/api.go

```diff
@@ -62,3 +62,19 @@ func (a *BlockValidatorDebugAPI) ValidationInputsAt(ctx context.Context, msgNum
 ) (server_api.InputJSON, error) {
 	return a.val.ValidationInputsAt(ctx, arbutil.MessageIndex(msgNum), target)
 }
+
+type MaintenanceAPI struct {
+	runner *MaintenanceRunner
+}
+
+func (a *MaintenanceAPI) GetLastMaintenanceTime(ctx context.Context) (string, error) {
```

Comment:
WDYT of this API returning, in a single call, the time in which last maintenance executed, current timestamp, and if maintenance is currently running?

Reply:
we don't really need an API for last maintenance.. we just need an API that returns error if we're in maintenance and I wanted it to return something reasonable otherwise.
Sort of following your direction I changed it to return seconds since last maintenance if not currently running, which does make more sense